packages/jbrowse-web/src/Loader.tsx

```diff
@@ -24,10 +24,75 @@ const useStyles = makeStyles({
   },
 })
 
+function NoConfigMessage() {
+  // TODO: Link to docs for how to configure JBrowse
```

Comment:
This is very interesting, why not remove the inDevelopment aspect of this though?

Reply:
I think in prod we could maybe display one or two small built-in example configs, but I wouldn't want all our testing configs to show up in a prod build.